apps/emqx_connector/src/emqx_connector_lib.erl

```diff
@@ -0,0 +1,22 @@
+%%--------------------------------------------------------------------
+%% Copyright (c) 2022 EMQ Technologies Co., Ltd. All Rights Reserved.
+%%
+%% Licensed under the Apache License, Version 2.0 (the "License");
+%% you may not use this file except in compliance with the License.
+%% You may obtain a copy of the License at
+%%
+%%     http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing, software
+%% distributed under the License is distributed on an "AS IS" BASIS,
+%% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+%% See the License for the specific language governing permissions and
+%% limitations under the License.
+%%--------------------------------------------------------------------
+-module(emqx_connector_lib).
+
+-export([resolve_dns/2]).
+
+%% @doc Mostly for meck.
```

Comment:
Q: using the `unstick` option doesn't work?

Reply:
It's not a sticky module.
The issue of mocking `inet_res` is it has too large of a scope which may cause the tests to become flaky.